pkg/config/validation/validation.go

```diff
@@ -482,9 +482,28 @@ var ValidateDestinationRule = registerValidateFunc("ValidateDestinationRule",
 		}
 
 		v := Validation{}
-		v = appendValidation(v,
-			ValidateWildcardDomain(rule.Host),
-			validateTrafficPolicy(rule.TrafficPolicy))
+		if features.EnableDestinationRuleInheritance {
+			if rule.Host == "" {
+				if len(rule.Subsets) != 0 {
+					v = appendValidation(v,
+						fmt.Errorf("mesh/namespace destination rule cannot have subsets"))
+				}
+				if len(rule.ExportTo) != 0 {
+					v = appendValidation(v,
+						fmt.Errorf("mesh/namespace destination rule cannot have exportTo configured"))
```

Comment:
Is this true?  What about value is . or *?  see https://istio.io/latest/docs/reference/config/networking/destination-rule/#DestinationRule

Reply:
Also, solo is adding support to Istio to allow mesh admin to configure certain namespaces to be part of Istio mesh.  The exportTo configure at global or namespace DR could be useful.

Reply:
ok with resolving this in a follow up PR